glib/src/auto/functions.rs

```diff
@@ -196,6 +196,73 @@ pub fn dpgettext2(domain: Option<&str>, context: &str, msgid: &str) -> crate::GS
     }
 }
 
+#[doc(alias = "g_environ_getenv")]
+pub fn environ_getenv(
+    envp: &[&std::path::Path],
+    variable: impl AsRef<std::path::Path>,
+) -> Option<std::path::PathBuf> {
+    unsafe {
+        from_glib_none(ffi::g_environ_getenv(
+            envp.to_glib_none().0,
+            variable.as_ref().to_glib_none().0,
+        ))
+    }
+}
+
+#[doc(alias = "g_environ_setenv")]
+pub fn environ_setenv(
+    envp: &[&std::path::Path],
+    variable: impl AsRef<std::path::Path>,
+    value: impl AsRef<std::path::Path>,
+    overwrite: bool,
+) -> Vec<std::path::PathBuf> {
+    unsafe {
+        FromGlibPtrContainer::from_glib_full(ffi::g_environ_setenv(
+            envp.to_glib_full(),
+            variable.as_ref().to_glib_none().0,
+            value.as_ref().to_glib_none().0,
+            overwrite.into_glib(),
+        ))
+    }
+}
+
+#[doc(alias = "g_environ_unsetenv")]
+pub fn environ_unsetenv(
+    envp: &[&std::path::Path],
+    variable: impl AsRef<std::path::Path>,
+) -> Vec<std::path::PathBuf> {
+    unsafe {
+        FromGlibPtrContainer::from_glib_full(ffi::g_environ_unsetenv(
+            envp.to_glib_full(),
+            variable.as_ref().to_glib_none().0,
+        ))
+    }
+}
+
+#[doc(alias = "g_file_get_contents")]
+pub fn file_get_contents(filename: impl AsRef<std::path::Path>) -> Result<Vec<u8>, crate::Error> {
```

Comment:
This should stay manual because it uses `Slice<u8>` instead of `Vec<u8>`. At a later time this can be autogenerated once I added support for that to gir.